modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,257 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
 ======
 =====
 
+=== Access Neo4j on port `:80` or `:443`
+
+The Neo4j Helm charts provide a Helm chart that allows you to use a Kubernetes Ingress to access Neo4j on port `:80` or `:443`.
+The Helm chart is called _neo4j/neo4j-reverse-proxy_ and is available on the Neo4j Helm repository from version 5.12.0.
+For more information about Kubernetes Ingress, see the link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes official documentation -> Ingress].
+
+The Helm chart creates a reverse proxy that is configured to route traffic to the Neo4j service URL using the `serviceName`, `namespace`, and `domain` values.
+For example, if the `serviceName` is _standalone-admin_, the `namespace` is _default_, and the `domain` is _cluster.local_, then the Neo4j service URL is _standalone-admin.default.svc.cluster.local_. +
+For Neo4j clusters, the Neo4j headless service is used to route the traffic to the cluster instances.
+For more information and a detailed example of how to install the _neo4j/neo4j-cluster-headless-service_ Helm chart, see xref:kubernetes/quickstart-cluster/access-inside-k8s.adoc#cc-access-headless[Access the Neo4j cluster using headless service].
+
+image::reverse-proxy.svg[title="Reverse proxy flow diagram"]
+
+The reverse proxy creates an HTTP server, which routes requests to either the Bolt reverse proxy or HTTP reverse proxy based on the request headers.
+Upon receiving a response, the Bolt reverse proxy updates the response to replace the Bolt port with either `:80` or `:443`.
+
+==== Configuration options
+
+To see all configurable options, run the following command:
+
+[source, shell]
+--
+helm show values neo4j/neo4j-reverse-proxy
+--
+
+[source, yaml]
+----
+# Default values for neo4j reverse proxy helm chart
+
+## @param nameOverride String to partially override common.names.fullname
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+fullnameOverride: ""
+
+# Parameters for reverse proxy
+reverseProxy:
+  image: "neo4j/neo4j-reverse-proxy"
```

Comment:
we can update the image to neo4j/helm-charts-reverse-proxy:5.12.0

Reply:
Good catch!

Reply:
Ah, I thought that this one would follow the same naming convention as the other helm charts. I'll rename it everywhere to helm-charts-reverse-proxy.